README.md

```diff
@@ -40,70 +65,28 @@ you can also have a look at [GitHub's getting started documentation][github-star
-but it will be quite inconvenient when working on single exercises.
+In order to contribute code to this track, you will probably want
+[npm][npm-install], [elm][elm-install], [elm-test][elm-test], and [elm-format][elm-format] installed globally.
+
+The build and test script for this track lives at `bin/build.sh`, and use npx, so can work without the rest of the tools being installed if required.
 
 [npm-install]: https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
 [elm-install]: https://guide.elm-lang.org/install/elm.html
 [elm-test]: https://www.npmjs.com/package/elm-test
 [elm-format]: https://github.com/avh4/elm-format
 
-### Preparing for v3 of Exercism
-
-If you are interesting in investing mid-term and long-term energy into this project,
-have a look at [issue #280][v3-issue].
+### Adding Missing Concept Exercise
 
-[v3-issue]: https://github.com/exercism/elm/issues/280
+Version 3 of Exercism introduced Concepts and Concept Exercises, which are a completely new thing. There is a [dependency diagram](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggTFJcbiAgQmFzaWNzW01pbmltYWwgY29tcGlsYWJsZSBmaWxlXSAtLT4gSW5kZXBlbmRlbnRzW0ltcG9ydCwgRXhwb3NpbmcsIEZ1bmN0aW9ucywgRmxvYXQsIEludCwgVHlwZSBBbm5vdGF0aW9uc10gLS0-IE1hdGhlbWF0aWNhbC1PcGVyYXRvcnNcbiAgSW5kZXBlbmRlbnRzIC0tPiBNYXRoZW1hdGljYWwtT3BlcmF0b3JzXG4gIEluZGVwZW5kZW50cyAtLT4gTWF0aGVtYXRpY2FsLUZ1bmN0aW9uc1xuICBJbmRlcGVuZGVudHMgLS0-IExldC1FeHByZXNzaW9uc1xuICBJbmRlcGVuZGVudHMgLS0-IEVxdWFsaXR5XG4gIEVxdWFsaXR5IC0tPiBPcmRlcmluZ1xuICBPcmRlcmluZyAtLT4gQ29tcGFyaXNvbnNcbiAgSW5kZXBlbmRlbnRzIC0tPiBTdHJpbmdcbiAgU3RyaW5nIC0tPiBSZWdleFxuICBJbmRlcGVuZGVudHMgLS0-IENoYXJcbiAgSW5kZXBlbmRlbnRzIC0tPiBMaXN0XG4gIExpc3QgLS0-IExpc3QtRXh0cmFcbiAgSW5kZXBlbmRlbnRzIC0tPiBEaWN0XG4gIEluZGVwZW5kZW50cyAtLT4gU2V0XG4gIEluZGVwZW5kZW50cyAtLT4gQXJyYXlcbiAgSW5kZXBlbmRlbnRzIC0tPiBUeXBlLWFsaWFzXG4gIEluZGVwZW5kZW50cyAtLT4gU3VtLXR5cGVzXG4gIEluZGVwZW5kZW50cyAtLT4gQm9vbGVhbnNbQm9vbCB0eXBlIC8gb3BlcmF0b3JzLCBJZl1cbiAgU3VtLXR5cGVzIC0tPiBQYXR0ZXJuLW1hdGNoaW5nXG4gIFBhdHRlcm4tbWF0Y2hpbmcgLS0-IE1heWJlXG4gIEJvb2xlYW5zIC0tPiBNYXliZVxuICBNYXliZSAtLT4gUmVzdWx0XG4gIFBhdHRlcm4tbWF0Y2hpbmcgLS0-IFBhcnNlclxuICBJbmRlcGVuZGVudHMgLS0-IEZ1bmN0aW9uLWNvbXBvc2l0aW9uXG4gIEluZGVwZW5kZW50cyAtLT4gRnVuY3Rpb24tY2hhaW5pbmdcbiAgSW5kZXBlbmRlbnRzIC0tPiBQYXJ0aWFsLWFwcGxpY2F0aW9uXG4gIFBhcnRpYWwtYXBwbGljYXRpb24gLS0-IFBvaW50LWZyZWVzdHlsZVxuICBGdW5jdGlvbi1jb21wb3NpdGlvbiAtLT4gUG9pbnQtZnJlZXN0eWxlXG4gIEZ1bmN0aW9uLWNoYWluaW5nIC0tPiBQb2ludC1mcmVlc3R5bGVcbiAgUG9pbnQtZnJlZXN0eWxlIC0tPiBPcGVyYXRvci1mdW5jdGlvbnNcbiAgSW5kZXBlbmRlbnRzIC0tPiBCaXR3aXNlLW9wZXJhdG9yc1xuICBJbmRlcGVuZGVudHMgLS0-IENvbnZlcnNpb25zXG4gIEluZGVwZW5kZW50cyAtLT4gUG9zaXgtdGltZSIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9) showing all the Elm concepts. You can see all concepts currently defined in [`concepts/`](concepts/), and you can se all concept exercises defined in [`/exercises/concept`](/exercises/concept).
```

Comment:
I put in a link to the diagram, as inline mermaid requires a plug in which not everyone has, and because it didn't work in the last time we used it, for some reason I didn't understand.